avm/res/service-bus/namespace/topic/main.bicep

```diff
@@ -57,18 +57,7 @@ param enablePartitioning bool = false
 param enableExpress bool = false
 
 @description('Optional. Authorization Rules for the Service Bus Topic.')
-param authorizationRules array = [
```

Comment:
Hey @nmcguffey, sorry for the delay in this review. I hope the module owner will forgive me if I'm chiming in.

- Please be aware that as per the [AVM contribution guidelines](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/), you should run the Set-AVMModule utility as detailed [here](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/generate-bicep-module-files/). You will see both the main.json and the README.md files will be updated by that.
Those updates are required for the validation pipeline to succeed and the contribution to be published.
- Also please consider updating the minor version of this product. Updating a default value would actually be a breaking change (major update) but since the module version is not yet at 1.0.0, we are only updating minor for both breaking changes and feature updates. 

Please reach out if any support needed.